src/components/Homepage/CategoryGrid.vue

```diff
@@ -0,0 +1,192 @@
+<template>
+	<div class="row text-center frame">
+		<h1 class="columns small-12 impact-text heading-text">
+			Browse loans by category
+		</h1>
+		<div class="row card-row">
+			<router-link
+				:to="loanChannel.url"
+				v-for="loanChannel in loanChannels" :key="loanChannel.id"
+				class="small-6 medium-4"
+				v-kv-track-event="[
+					'homepage',
+					'click-Category-card',
+					`${loanChannel.name}-category-card`,
+					'true']"
+			>
+				<div class="category-card-overlay">
+					<div>
```

Comment:
I think this div can be dropped since the .category-card-overlay div is already wrapping it